conans/client/cmd/export.py

```diff
@@ -99,6 +99,19 @@ def _capture_export_scm_data(conanfile, conanfile_dir, destination_folder, outpu
 def _replace_scm_data_in_conanfile(conanfile_path, scm_data):
     # Parsing and replacing the SCM field
     content = load(conanfile_path)
+    headers = []
+
+    if six.PY2:
+        # Workaround for https://bugs.python.org/issue22221
+        lines_without_headers = []
+        lines = content.splitlines(True)
+        for line in lines:
```

Comment:
It is not enough to remove the ``# encoding`` line?

Reply:
I put it later in the file to be able to parse/evaluate it... it can have non-ascii chars (and it reminds me that I need to add that in the test file! Thanks!)

Reply:
it already has non-ascii chars... there are several ``Ñandú!`` as authors... 😆